### PR TITLE
[java/pt] Some translation fixes.

### PR DESCRIPTION
**Pull request by YuriAlessandro**
- [x] PR touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at _intermediate to experienced programmers_ (this is a poor format for explaining fundamental programming concepts)
- [x] YAML Frontmatter formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Seriously, look at it now. Watch for quotes and double-check field names.

[java/pt] Improving the consistency of a few sentences.

### CODE REVIEW

**Comment by aloisdg** [comment]
Can you add `[java/pt]`? it would help :)

**Comment by YuriAlessandro** [comment]
@aloisdg Done! :smile:

**Comment by aloisdg** [comment]
Great!

File: pt-br/java-pt.html.markdown [hunk 1]
```diff
@@ -7,6 +7,7 @@ contributors:
 translators:
     - ["Victor Kléber Santos L. Melo", "http://victormelo.com.br/blog"]
     - ["Renê Douglas N. de Morais", "mailto:rene.douglas.bsi@gmail.com"]
+    - ["Yuri Alessandro D. T. Martins", "https://github.com/YuriAlessandro"]
```
Comment (vendethiel) [inline pt-br/java-pt.html.markdown]: LGTM, but we usually reserve the contributor tags to more significant contributions (like a second PR, a few paragraphs, ...)

**Pull request merged**